### PR TITLE
[USING] update macOS install

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch is updating the install instruction of MacOS. The PGP verification is moved in an optional drop down menu, so to not clutter the page. I have never used mac, and am not sure if this is actually correct. So please someone verify.

### CODE REVIEW

File: docs/using-wasabi/InstallPackage.md [hunk 1]
```diff
@@ -122,14 +122,30 @@ Among others, here is where your wallet files and your logs reside.
 
 ## macOS
 
+1. [Download](https://wasabiwallet.io/#download) the `.dmg` package of the latest Wasabi release ${currentVersion}.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/InstallPackage.md:125]:
```suggestion
1. [Download](https://wasabiwallet.io/#download) the `.dmg` package of Wasabi ${currentVersion}.
```

File: docs/using-wasabi/InstallPackage.md [hunk 2]
```diff
@@ -122,14 +122,30 @@ Among others, here is where your wallet files and your logs reside.
...
+
+![](/DownloadMac.png)
+
+2. Double click `.dmg` to open it.
+
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/InstallPackage.md:130]:
```suggestion
```

File: docs/using-wasabi/InstallPackage.md [hunk 3]
```diff
@@ -122,14 +122,30 @@ Among others, here is where your wallet files and your logs reside.
...
+3. Install Wasabi by dragging it into your `Applications` folder.
+
+![](/InstallMacDragDrop.png)
+
+4. At first startup, there will be a pop-up that Wasabi was downloaded from the internet.
+Click on `Open` and restart Wasabi.
+
+::::details
+### Optional PGP Verification
+
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/InstallPackage.md:140]:
```suggestion
```

File: docs/using-wasabi/InstallPackage.md [hunk 4]
```diff
@@ -122,14 +122,30 @@ Among others, here is where your wallet files and your logs reside.
...
 If you have already imported zkSNACKs' PGP public key, then jump to step 4.
 
 1. [Get GnuPG](https://www.gnupg.org/download/index.html).
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/InstallPackage.md:144]:
```suggestion
```

File: docs/using-wasabi/InstallPackage.md (hunk 2 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/InstallPackage.md:130]: NACK, there is always a line here

File: docs/using-wasabi/InstallPackage.md (hunk 4 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/InstallPackage.md:144]: NACK, there is always a line here

File: docs/using-wasabi/InstallPackage.md (hunk 3 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/InstallPackage.md:140]: NACK, there is always a line here

**Comment by molnard** [comment]
Very good!

**Pull request merged**